core/controllers/classroom.py

```diff
@@ -139,23 +139,55 @@ def get(self, classroom_url_fragment: str) -> None:
         self.render_json(self.values)
 
 
-class ClassroomIdToNameHandler(
```

Comment:
I don't think there's a need to rename the handler and mention the new field in every place. Just add a new field in the returning object, maybe in comments, but definitely don't change the handler name and the objects' names. Otherwise, when we need to add another field, we'd have to rename everything again.